org.mixedrealitytoolkit.input/Experimental/XRI3/HandModel.cs

```diff
@@ -56,6 +57,15 @@ public class HandModel : MonoBehaviour
         /// <remarks>Expected to be XRNode.LeftHand or XRNode.RightHand.</remarks>
         public XRNode HandNode => handNode;
 
+        [SerializeField, Tooltip("The select input reader to be used by the hand model prefab when implementing ISelectInputVisualizer.")]
```

Comment:
As per https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk2/contributing/coding-guidelines?view=mrtkunity-2022-05#do-10
```suggestion
        [SerializeField, Tooltip("The XRInputButtonReader representing selection values to be used by the hand model prefab when implementing ISelectInputVisualizer.")]
```